File: 268-MissingNumber/268-MissingNumber.py
# Last updated: 5/29/2026, 11:56:36 AM
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        nums.sort()
        for i in range(nums[len(nums)-1]):
            if i!=nums[i]:
                return i
        return nums[-1]+1

File: 268-MissingNumber/268-MissingNumber.java
// Last updated: 5/29/2026, 11:56:41 AM
public class Solution {
    public int missingNumber(int[] nums) {
        int n = nums.length;
        HashSet<Integer> hs = new HashSet<>();
        for (int i = 0; i <= n; i++) {
            hs.add(i);
        }
        for (int num : nums) {
            hs.remove(num);
        }
        return hs.iterator().next();
    }
}
